api/Navigator.json

```
@@ -727,7 +727,9 @@
               "version_added": false
             },
             "edge": {
-              "version_added": "81"
+              "version_added": "83",
+              "partial_implementation": true,
+              "notes": "Windows and Mac only."
```

```suggestion
              "notes": "Windows and macOS only."
```

Can you land https://github.com/mdn/browser-compat-data/pull/14222?